clusterloader2/testing/load/config.yaml

```diff
@@ -77,6 +77,8 @@
 
 # Command to be executed
 {{$EXEC_COMMAND := DefaultParam .CL2_EXEC_COMMAND nil}}
+{{$EXIT_AFTER_EXEC := DefaultParam .CL2_EXIT_AFTER_EXEC false}}
+{{$WAIT_AFTER_EXEC_DURATION := DefaultParam .CL2_WAIT_AFTER_EXEC_DURATION "0s"}}
```

Comment:
WAIT_AFTER_EXEC_DURATION is misleading, because don't wait for anything we just sleep.

should probably be SLEEP_AFTER_EXEC_DURATION

Reply:
Sure, I will fix it.